### PR TITLE
[JA] Translate error-handling 1.1.0

### PR DESCRIPTION
**Pull request by starfish719**
I translated `lessons/advanced/error-handling` section.
Please review! @elixirschool/japanese

refs:
https://github.com/elixirschool/elixirschool/commit/8a8f948636d464505a47856cd05728504fe30d78#diff-1c4ec0de66a685785dc9f2d4074b730c
https://github.com/elixirschool/elixirschool/commit/1a4fc79a280b6ef91f8e6fa77eb2add96d8b47d3#diff-1c4ec0de66a685785dc9f2d4074b730c
https://github.com/elixirschool/elixirschool/commit/696f1ee54f332134340e18428fe5ee45b55e6d49#diff-1c4ec0de66a685785dc9f2d4074b730c

### CODE REVIEW

File: ja/lessons/advanced/error-handling.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ title: エラーハンドリング
...
+## 一般的な規則
+
+現時点では、Elixirコミュニティにはエラーを返すことに関していくつかの慣習があります:
+
+* 関数の通常の操作の一部であるエラー（たとえば、ユーザーが間違ったタイプの日付を入力した場合）の場合、関数はそれに応じて `{：ok、result}` と `{：error、reason}` を返します。
+* 通常の操作の一部ではないエラー（構成データを解析できないなど）の場合は、例外をスローします。
+
+通常、標準エラーは[パターンマッチング](../../basics/pattern-matching/)で処理しますが、このレッスンでは、例外について2番目のケースに焦点を当てます。
+
+多くの場合、パブリックAPIでは、ラップされていない結果を返すか、エラーを発生させる ！（example！/ 1）を使用して関数の2番目のバージョンを見つけることもできます。
```
Comment (marocchino) [inline ja/lessons/advanced/error-handling.md:23]:
```suggestion
多くの場合、パブリックAPIでは、ラップされていない結果を返すか、エラーを発生させる ！（example!/1）を使用して関数の2番目のバージョンを見つけることもできます。
```

Comment (starfish719) [reply ja/lessons/advanced/error-handling.md]: 👍

File: ja/lessons/advanced/error-handling.md [hunk 2]
```diff
@@ -11,6 +11,17 @@ title: エラーハンドリング
 
 {% include toc.html %}
 
+## 一般的な規則
+
+現時点では、Elixirコミュニティにはエラーを返すことに関していくつかの慣習があります:
+
+* 関数の通常の操作の一部であるエラー（たとえば、ユーザーが間違ったタイプの日付を入力した場合）の場合、関数はそれに応じて `{：ok、result}` と `{：error、reason}` を返します。
```
Comment (marocchino) [inline ja/lessons/advanced/error-handling.md:18]:
```suggestion
* 関数の通常の操作の一部であるエラー（たとえば、ユーザーが間違ったタイプの日付を入力した場合）の場合、関数はそれに応じて `{:ok、result}` と `{:error、reason}` を返します。
```

File: ja/lessons/advanced/error-handling.md [hunk 3]
```diff
@@ -11,6 +11,17 @@ title: エラーハンドリング
...
+* 通常の操作の一部ではないエラー（構成データを解析できないなど）の場合は、例外をスローします。
+
+通常、標準エラーは[パターンマッチング](../../basics/pattern-matching/)で処理しますが、このレッスンでは、例外について2番目のケースに焦点を当てます。
```
Comment (marocchino) [inline ja/lessons/advanced/error-handling.md:21]:
```suggestion
通常、標準エラーは[パターンマッチング](../../basics/pattern-matching/)で処理しますが、このレッスンでは、2番目のケース、つまり例外について焦点を当てます。
```

File: ja/lessons/advanced/error-handling.md [hunk 4]
```diff
@@ -11,6 +11,17 @@ title: エラーハンドリング
...
+
+多くの場合、パブリックAPIでは、ラップされていない結果を返すか、エラーを発生させる ！（example!/1）を使用して関数の2番目のバージョンを見つけることもできます。
```
Comment (marocchino) [inline ja/lessons/advanced/error-handling.md:23]:
```suggestion
大体のパブリックAPIでは、!がついている関数（example!/1）かを見て後者かを確認することもできます。こいう関数はラップされていない結果を返すか、エラーを発生させます。
```

> Often, in public APIs, you can also find a second version of the function with a !  that returns the unwrapped result or raises an error.

**Review by marocchino — APPROVED** [review]

File: ja/lessons/advanced/error-handling.md [hunk 5]
```diff
@@ -20,7 +20,7 @@ title: エラーハンドリング
 
 通常、標準エラーは[パターンマッチング](../../basics/pattern-matching/)で処理しますが、このレッスンでは、2番目のケース、つまり例外について焦点を当てます。
 
-大体のパブリックAPIでは、!がついている関数（example!/1）かを見て後者かを確認することもできます。こいう関数はラップされていない結果を返すか、エラーを発生させます。
+大体のパブリックAPIでは、!がついている関数（example!/1）かを見て後者かを確認することもできます。このような関数はラップされていない結果を返すか、エラーを発生させます。
```
Comment (marocchino) [inline ja/lessons/advanced/error-handling.md:23]: 👍

**Pull request merged**